err-standup/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pytz==2016.7
+PyYAML==3.12
```

Comment:
I just did a 'pip freeze > requirements.txt'. What's a better approach? I'm pretty new at creating full packages in python still.